packages/site/src/lib/helpers/entry/displayFields.test.ts

```diff
@@ -0,0 +1,61 @@
+import type { IEntry } from '@living-dictionaries/types';
+import {
+  showLocalOrthographies,
+  showPartsOfSpeech,
+  removeItalicTagsWithAPeriod,
+  unnecessaryLineBreaksRegex,
+} from './displayFields';
+import { entriesWithAlternateOrthographies } from '../../../routes/[dictionaryId]/entries/print/mock-data';
```

Comment:
I think we should move the mock-data file into a more general folder, what'd you think @jacob-8